frontend/src/app/organization/organization.resolver.ts

```diff
@@ -34,7 +39,8 @@ export const organizationDetailResolver: ResolveFn<Organization | undefined> = (
             linked_in: "",
             youtube: "",
             heel_life: "",
-            public: false
-          };
+            public: false,
+            events: null
```

Comment:
Should this be an empty array rather than null?

Reply:
@KrisJordan I think by leaving it null, it matches the shape of the `Event` backend model. May be wrong though